server/http_handler_test.go

```diff
@@ -1228,6 +1228,8 @@ func (ts *HTTPHandlerTestSuite) TestPostSettings(c *C) {
 	c.Assert(resp.StatusCode, Equals, http.StatusOK)
 	c.Assert(config.GetGlobalConfig().CheckMb4ValueInUTF8, Equals, false)
 	dbt.mustExec("insert t2 values (unhex('f09f8c80'));")
+	// restore original value.
+	config.GetGlobalConfig().CheckMb4ValueInUTF8 = true
```

Comment:
I think test cases modifying global config better go to serial tests. As it may cause data race.